### DISCUSSION TITLE
Debug Template Tags: Show Site name/ID in results

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#3219] [Feature Requests]
The Debug Template Tags utility is great!

However, it reports template tag usage in _any_ template, even templates that aren't accessible for the logged_in user depending on their MSM assignment.

So I've found an offending tag, clicked on the template from the modal window, only to find I couldn't access it.

**Comment by intoeetive** [comment]
Classifying as feature request (also quick investigation showed this would require some rework of how we fetch the data)
